deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,71 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    background-color: #F2F3D9;
+    font-family: 'Poppins', sans-serif;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    background-color: #151E3F;
+    padding: 40px;
+    justify-content: space-between;
+}
+
+.navUL {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    list-style: none;
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+.anchor ~ a {
+    justify-content: flex-end;
```

Comment:
Dit is volgens mij niet nodig. Met de space between - die je op je nav hebt staan - heb je het al geregeld.